docs/source/install.rst

```diff
@@ -44,16 +39,7 @@ Nivio expects a seed configuration at start time (unless you want to run the dem
 
 then point your browser to the GUI at http://localhost:8080 or the API at http://localhost:8080/api/.
 
-Behind a proxy
---------------
-
-If you deploy nivio to run under a different path than root ("/"), make sure to set the environment variables
-SERVER_CONTEXT_PATH and NIVIO_BASEURL to the path.
-
-.. code-block:: bash
-
-   SERVER_SERVLET_CONTEXT_PATH: /my-landscape
-   NIVIO_BASEURL: https://foo.com/my-landscape/
+There is a demo project "nivio-demo" in the repository under `nivio/nivio-demo`, which starts an nginx to simulate a remote server.
```

Comment:
Reading this, I still don't understand 100% how this simulates a remote server. So it simulates a remote server for the seed files? (We should write that out clearly.)

Is this the reason why my attempts to use that `docker-compose.yml` with a local file don't really work?

Reply:
I've pushed another version of the file. The demo is meant to simulate nivio running in a docker container without mounting files directly into it, and using nginx to serve configs instead.

If you have any better ideas, let me know.